_includes/landing-page/dockercon-banner.html

```diff
@@ -2,16 +2,15 @@
   <div class="container">
     <div class="row">
       <div class="col-xs-12 col-md-6 col-lg-offset-1 col-lg-7">
-        <img src="/images/dockercon21.png" alt="dockercon" />
```

Comment:
Oh! can you perhaps also remove the `dockercon21.png` image in this PR, as it's no longer used? https://github.com/docker/docker.github.io/blob/master/images/dockercon21.png

Reply:
Good point. I've removed the old image